tests/platform/mellanox/check_sysfs.py

```diff
@@ -77,3 +77,30 @@ def check_sysfs(dut):
     for fan_speed_set in fan_speed_set_list:
         fan_speed_set_content = dut.command("cat %s" % fan_speed_set)
         assert fan_speed_set_content["stdout"] == "153", "Fan speed should be set to 60%, 153/255"
+
+def check_psu_sysfs(dut, psu_id, psu_state):
+    """
+    @summary: Check psu related sysfs under /var/run/hw-management/thermal against psu_state
+    """
+    psu_exist = "/var/run/hw-management/thermal/psu%s_status" % psu_id
+    if psu_state == "NOT PRESENT":
+        psu_exist_content = dut.command("cat %s" % psu_exist)
+        logging.info("psu state %s file %s read %s" % (psu_state, psu_exist, psu_exist_content["stdout"]))
+        assert psu_exist_content["stdout"] == "0", "cli return NOT PRESENT while %s contains %s" %  \
```

Comment:
'cli return**s**' 
should we use CLI and PSU in any logging message? in some i do see PSU and in some psu. Lets use same way for all.